save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/userprofile/UserProfileView.kt

```diff
@@ -371,6 +373,42 @@ fun ChildrenBuilder.renderLeftUserMenu(
     }
 }
 
+/**
+ * @param name
+ * @param header
+ */
+fun ChildrenBuilder.validateName(name: String?, header: HTMLAttributes<HTMLHeadingElement>) {
+    name?.let {
+        if (it.length > 14) {
+            asDynamic()["data-toggle"] = "tooltip"
+            asDynamic()["data-placement"] = "bottom"
+            header.title = it
+
+            +(it.substring(0, 13) + "...")
```

Comment:
there is a file `save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/StringUtils.kt`
can you please add this method there?

Reply:
Done